workers-site/index.js

```diff
@@ -1,17 +1,16 @@
-import { getAssetFromKV } from "@cloudflare/kv-asset-handler";
+import { getAssetFromKV } from '@cloudflare/kv-asset-handler'
 
-addEventListener("fetch", event => {
-  event.respondWith(handleEvent(event));
-});
+addEventListener('fetch', event => {
+  event.respondWith(handleEvent(event))
+})
 
 async function handleEvent(event) {
   try {
-    return await getAssetFromKV(event);
+    return await getAssetFromKV(event)
   } catch (e) {
-    let pathname = new URL(event.request.url).pathname;
-    return new Response(`"${pathname}" not found`, {
+    let pathname = new URL(event.request.url).pathname
+    return new Response(e.message || e.toString(), {
       status: 404,
-      statusText: "not found"
```

Comment:
note `statusText` is set when one sets `status` to the default HTTP status definition of whatever HTTP code